static/js/components/tiles/ranking_tile.tsx

```diff
@@ -54,6 +54,145 @@ interface RankingGroup {
   dateRange: string;
 }
 
+interface SvRankingUnitsProps {
+  dateRange: string;
+  numDataPoints: number;
+  highestTitle?: string;
+  isMultiColumn?: boolean;
+  lowestTitle?: string;
+  points: RankingPoint[];
+  scaling: number[];
+  showHighest: boolean;
+  showLowest: boolean;
+  rankingCount: number;
+  showChart: (
```

Comment:
nit: showChartEmbed or something like that? showChart makes me think this is just to show the chart in the tile

Reply:
done